### PR TITLE
add missing route to streams lineage in RouterPathLookup

### PR DESCRIPTION
**Pull request by gsps1**
we were missing the route to streams lineage.

### CODE REVIEW

File: cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java [hunk 1]
```diff
@@ -88,6 +88,7 @@ private String getV3RoutingService(String [] uriParts, AllowedMethod requestMeth
       matches(uriParts, "v3", "namespaces", null, "streams", null, "metadata", "tags") ||
       matches(uriParts, "v3", "namespaces", null, "metadata", "search") ||
       matches(uriParts, "v3", "namespaces", null, "datasets", null, "lineage") ||
+      matches(uriParts, "v3", "namespaces", null, "streams", null, "lineage") ||
```
Comment (bdmogal) [inline cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java]: add a verification for this in ``RouterPathTest.testMetadataRouting()`` as well.

Comment (gsps1) [inline cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java]: done

**Comment by bdmogal** [comment]
LGTM. Merge after successful build.

**Comment by gsps1** [comment]
Green, merging.

File: cdap-gateway/src/test/java/co/cask/cdap/gateway/router/RouterPathTest.java [hunk 2]
```diff
@@ -355,6 +355,7 @@ public void testMetadataPath() {
     assertMetadataRouting("/v3/namespaces/default/metadata/search");
     // lineage
     assertMetadataRouting("/v3/namespaces/default/////datasets/ds1/lineage");
+    assertMetadataRouting("/v3/namespaces/default/streams/st1/lineage");
```
Comment (poornachandra) [inline cdap-gateway/src/test/java/co/cask/cdap/gateway/router/RouterPathTest.java]: Does this test succeed? Looks like it is missing a few path elements.

Comment (gsps1) [inline cdap-gateway/src/test/java/co/cask/cdap/gateway/router/RouterPathTest.java]: it does pass, which path elements are missing ?

**Pull request merged**